### DISCUSSION TITLE
Python 3.8

### DISCUSSION THREAD

**Discussion by jbuch808** [PaddlePaddle/PaddleOCR#6409] [Q&A]
I was wondering if there are any plans to support Python 3.8? The M1 Mac does not support anything before 3.8 so it currently fails to install. I tried it out in Google Colab and it works great! I would love to run it natively so if there are any tips I would greatly appreciate it. Thanks!

**Comment by WenmuZhou** [comment]
On my mac m1 computer, python3.8 can install paddleocr, you can try to compile and install it yourself

**Comment by SUPERustam** [comment]
Are you can run PaddleOCR on Apple Silicon? I also have Apple Silicon: M2 and I want to run PaddleOCR on my machine, but have issues with compatibility with my chip architecture. In Github codespace all works fine, but it cost additional money:(

How to run PaddleOCR on Apple Silicon?

**Reply by SWHL** [reply]
Refer this [link](https://github.com/PaddlePaddle/PaddleOCR/discussions/13060?converting=1#discussioncomment-9762733)
